libdrgn/arch_s390_defs.py

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+REGISTERS = [
+    *[DrgnRegister(f"r{i}") for i in range(0,16)],
+    DrgnRegister("pswm"),
+    DrgnRegister("pswa"),
+    DrgnRegister("bc"),
+]
+
+REGISTER_LAYOUT = [
+    # Caller-saved registers.
+    *[DrgnRegisterLayout(f"r{i}", size=8, dwarf_number=i) for i in range(0,16)],
+    DrgnRegisterLayout("pswm", size=8, dwarf_number=64),
+    DrgnRegisterLayout("pswa", size=8, dwarf_number=65),
+    DrgnRegisterLayout("bc", size=8, dwarf_number=127),
+]
```

Comment:
Since you submitted this, I added the stack pointer register to the register definition format:

```suggestion
]

STACK_POINTER_REGISTER = "r15"
```